### PR TITLE
Gym info update

### PR DESCRIPTION
**Pull request by ClarkyKent**
This is a minor PR to update some visual and QOL improvements on gym bar info.

## Description
This PR brings 2 changes mainly

- Order pokemons by deployed time instead of decayed CP
- Shows decayed CP in three different color (green , orange and red) to indicate how many battles need to kick a pokemon out of gym. And original PC is added in drop down with IVs.

## Motivation and Context
Ordering by deployed time help is more usefull than ordering via CP decayed. Let you know correct order of Pokemon to battle.
And color coding of Decayed CP helps to know battles required at a glance.

## How Has This Been Tested?
personal map

## Screenshots (if appropriate):
rocketmap - mozilla firefox

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "main": "Gruntfile.js",
   "dependencies": {
     "babel-preset-env": "^1.6.0",
+    "cross-zip-cli": "^1.0.0",
```
Comment (pogo-excalibur) [inline package.json]: What's `cross-zip` offering here?

File: static/js/map.js [hunk 2]
```diff
@@ -2192,7 +2192,16 @@ function showGymDetails(id) { // eslint-disable-line no-unused-vars
 function getSidebarGymMember(pokemon) {
     var perfectPercent = getIv(pokemon.iv_attack, pokemon.iv_defense, pokemon.iv_stamina)
     var moveEnergy = Math.round(100 / pokemon.move_2_energy)
-
+    const motivationColor = ['16,155,49,1', '235,193,26,1', '255,25,25,1']
+    const motivationPercentage = (pokemon.cp_decayed / pokemon.pokemon_cp) * 100
+    var colorIdx = 0
+    if (motivationPercentage <= 46.66) {
```
Comment (pogo-excalibur) [inline static/js/map.js]:
```
var colorIdx = 0
if (motivationPercentage < 46.66) {
    colorIdx = 2
} else if (motivationPercentage < 73.33) {
    colorIdx = 1
}
```

File: package.json (hunk 1 above)
Comment (ClarkyKent) [reply package.json]: DO not know how this file got changed.. 
I did not change anything in there. I'll revert it.

File: static/js/map.js [hunk 3]
```diff
@@ -2201,7 +2210,7 @@ function getSidebarGymMember(pokemon) {
                         </td>
                         <td>
                             <div class="gym pokemon" style="line-height:0.5em;">${pokemon.pokemon_name}</div>
-                            <div><img class="gym pokemon motivation heart" src="static/images/gym/Heart.png"> <span class="gym pokemon motivation">${pokemon.cp_decayed}</span></div>
+                            <div><img class="gym pokemon motivation heart" src="static/images/gym/Heart.png"> <span class="gym pokemon motivation" style='color:rgba(${motivationColor[colorIdx]})'>${pokemon.pokemon_cp}</span></div>
```
Comment (123FLO321) [inline static/js/map.js]: Why did you change `pokemon.cp_decayed` to `pokemon.pokemon_cp` here? Is this indented?

Comment (ClarkyKent) [reply static/js/map.js]: No, it was not.

File: static/js/map.js [hunk 4]
```diff
@@ -2192,7 +2192,14 @@ function showGymDetails(id) { // eslint-disable-line no-unused-vars
 function getSidebarGymMember(pokemon) {
     var perfectPercent = getIv(pokemon.iv_attack, pokemon.iv_defense, pokemon.iv_stamina)
     var moveEnergy = Math.round(100 / pokemon.move_2_energy)
-
+    const motivationColor = ['16,155,49,1', '235,193,26,1', '255,25,25,1']
```
Comment (friscoMad) [inline static/js/map.js]: I would prefer to have this colors in CSS with classes and just use a class here, it is mostly the same but styles and color definitions should be outside the js.

**Comment by friscoMad** [comment]
Working ok and nice PR but add the suggested changes please.

File: static/js/map.js [hunk 5]
```diff
@@ -2192,16 +2191,28 @@ function showGymDetails(id) { // eslint-disable-line no-unused-vars
...
                         <td>
-                            <div class="gym pokemon" style="line-height:0.5em;">${pokemon.pokemon_name}</div>
-                            <div><img class="gym pokemon motivation heart" src="static/images/gym/Heart.png"> <span class="gym pokemon motivation">${pokemon.cp_decayed}</span></div>
+                            <div class="gym pokemon" style="line-height:1em;"><span class="gym pokemon name">${pokemon.pokemon_name}</span></div>
+                            <div>
+                            <span class="gym pokemon motivation decayed zone ${motivationZone[colorIdx].toLowerCase()}">${pokemon.cp_decayed}</span>
+                            </div>
+                            <div>
+                            <span class="gym pokemon motivation cp">Max: ${pokemon.pokemon_cp}</span>
+                            </div>
```
Comment (sebastienvercammen) [inline static/js/map.js]: Indent the spans a bit more.
